recipes/dcmstack/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version = "v0.6.2" %}
+
+package:
+  name: dcmstack
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/moloney/dcmstack.git
+  git_tag: {{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - dcmstack
+  commands:
+    - python -c "import example;"
```

Comment:
Yes - that's great! Comments like: "md5 is OK but sha256 is desirable." are exactly at the level I need. 

You might want to have several different options in there for each section, and comment out the less desirable options, explaining (for example) that you prefer tar-balls to git clones (in the `source` section), but here's how you would do either of these.